scripts/etc/init.d/rcS

```diff
@@ -760,7 +760,7 @@ cd /rootfs/boot/config/apt/
 # iterate through all the *.list files and add them to /etc/apt/sources.list.d
 for listfile in ./*.list
```

Comment:
of course, mainly because I always have additional checks, pretty much like in this case here